sub-pages/_single.html

```diff
@@ -24,35 +24,35 @@
     %}
 
     {% if sub_page.title %}
-    <h1>{{ sub_page.title | safe }}</h1>
+        <h1>{{ sub_page.title | safe }}</h1>
     {% endif %}
 
     {% if sub_page.content %}
-    <section class="block
-                    block__flush-top
-                    block__flush-bottom">
-        {{ sub_page.content | safe }}
-    </section>
+        <section class="block
+                        block__flush-top
+                        block__flush-bottom">
+            {{ sub_page.content | safe }}
+        </section>
     {% endif %}
 
     {% if sub_page.body_content %}
-    <section class="sub-page_{{sub_page.slug}}
-                    block
-                    block__padded-top
-                    block__border-top">
-        {% set content = { 'markup' : sub_page.body_content } %}
-        {% for form_name in vars.forms %}
-            {% set form_token = '[' + form_name + ']' %}
-            {% if (sub_page.body_content).find(form_token) > -1 %}
-                {% import 'templates/forms/' + form_name | lower  + '.html' as form %}
-                {% set _ignore = content.update(
-                    { 'markup' : sub_page.body_content | replace(form_token, form) }
-                )%}
-            {% endif %}
-        {% endfor %}
+        <section class="sub-page_{{sub_page.slug}}
+                        block
+                        block__padded-top
+                        block__border-top">
+            {% set content = { 'markup' : sub_page.body_content } %}
```

Comment:
Yeah-- soon we'll be able to start using back-end code to populate the template context, use more flexible URL patterns, etc.